braze/build.gradle

```diff
@@ -25,21 +26,30 @@ android {
         }
     }
 
+    buildFeatures {
+        buildConfig true
+    }
+
     compileOptions {
         sourceCompatibility JavaVersion.VERSION_11
         targetCompatibility JavaVersion.VERSION_11
     }
+
+    kotlinOptions {
+        jvmTarget = JavaVersion.VERSION_11
+    }
+
     namespace 'com.tealium.remotecommands.braze'
     testNamespace 'com.tealium.remotecommands.brazetest'
 }
 
 dependencies {
     implementation fileTree(include: ['*.jar'], dir: 'libs')
-    api 'com.tealium:remotecommands:1.0.1'
-    api 'com.braze:android-sdk-ui:[25.0.0,32.0)'
+    api 'com.tealium:remotecommands:1.0.3'
+    api 'com.braze:android-sdk-ui:[25.0.0,35.0)'
```

Comment:
some of these newly added methods look like they weren't added until after `25.0.0` so suggest we update that lower limit. shouldn't affect many people, but would be good to get this range correct for all the methods we use.